src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -1070,6 +1092,7 @@ def forward(self, inputs, targets):
         # METRICS
         self._set_train_metrics(train_metrics_list=self.training_params.train_metrics_list)
         self._set_valid_metrics(valid_metrics_list=self.training_params.valid_metrics_list)
+        self.test_metrics = self.valid_metrics.clone()
```

Comment:
Until we support custom test metrics, let me know if you want to do differently

Reply:
I'm not sure whether valid_metrics is MetricsCollection or generic list. 
So there is a chance that .clone() is doing a shallow copy which may not be what we are expecting.
Maybe copy.deepcopy() just to be safe?

Reply:
It's always a `MetricCollection` but it's not super explicit because `_set_train_metrics` does it inplace.

Reply:
Added typing here: https://github.com/Deci-AI/super-gradients/pull/1241/commits/2d51ab6cb000c0dc15728e1aeaeb90dc2840ed16